modelexpress_client/python/tests/test_reshard_refit_nixl_transport.py

```
@@ -0,0 +1,80 @@
+    descriptors = [
+        ReadDescriptor(session="sA", src_addr=1000, dst_addr=10, nbytes=16),
+        ReadDescriptor(session="sB", src_addr=2000, dst_addr=20, nbytes=8),
+        ReadDescriptor(session="sA", src_addr=1016, dst_addr=26, nbytes=16),
+    ]
+    transport.read(descriptors)
+
+    # One batched READ per session.
+    assert len(mgr.calls) == 2
+    by_agent = {c[0]: c for c in mgr.calls}
+
+    a_agent, a_ranges, a_mem, a_timeout = by_agent["trainer-agent-A"]
```

Ruff (0.15.21)

[warning] Unpacked variable `a_agent` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)